broker/thinBroker.go

```diff
@@ -1659,10 +1660,10 @@ func (tb *ThinBroker) removeFiwareHeadersFromId(ctxElem *ContextElement, fiwareS
 
 // Create an NGSI-LD Entity
 func (tb *ThinBroker) LDCreateEntity(w rest.ResponseWriter, r *rest.Request) {
-	var context []interface{}
-
+	fmt.Println("Inside LDCreateEntity..........")
 	//Also allow the header to json+ld for specific cases
 	if ctype, accept := r.Header.Get("Content-Type"), r.Header.Get("Accept"); (ctype == "application/json" || ctype == "application/ld+json") && accept == "application/ld+json" {
+		var context []interface{}
```

Comment:
[Major]default context error: in case of using w3.org context (https://www.w3.org/ns/json-ld#context)

Reply:
added support